app/frontend/styles/_filter.scss

```diff
@@ -30,6 +30,10 @@
   box-shadow: none;
 }
 
+.moj-pagination__list {
+  padding-left: 3px;
```

Comment:
Why is this needed? (It's not on the prototype.)

Reply:
For some reason without this it is the chevrons don't fit correctly.